hack/testdata/replicaset-with-initializer.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1beta2
 kind: ReplicaSet
 metadata:
```

Comment:
Do we want to add `label` field here? Adding the field is not strictly needed, but doing so makes this YAML file more consistent with others.